src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -27,14 +27,11 @@ abstract class KernelTestCase extends TestCase
     use MailerAssertionsTrait;
     use NotificationAssertionsTrait;
 
-    protected static $class;
+    protected static ?string $class;
 
-    /**
-     * @var KernelInterface
-     */
-    protected static $kernel;
+    protected static ?KernelInterface $kernel;
```

Comment:
```suggestion
    protected static ?KernelInterface $kernel = null;
```

Otherwise `ensureKernelShutdown()` will trip an uninitialized property error if the kernel hasn't been created.